frontend/src/forms/manage-restaurant-form/ImageSection.tsx

```diff
@@ -9,18 +10,27 @@ import { Input } from '@/components/ui/input';
 import { useFormContext } from 'react-hook-form';
 
 const ImageSection = () => {
-  const { control } = useFormContext();
+  const { control, watch } = useFormContext();
+  const existingImageUrl = watch('imageUrl');
 
   return (
     <div className="space-y-2">
       <div>
         <h2 className="text-2xl font-bold">Image</h2>
         <FormDescription>
-          Add a image that will be displayed on your resataurant listing
+          Add a image that will be displayed on your restaurant listing
         </FormDescription>
       </div>
 
-      <div className="flex flex-col gap-8 w-[50%]">
+      <div className="flex flex-col gap-8 md:w-[50%]">
+        {existingImageUrl && (
+          <AspectRatio ratio={16 / 9}>
+            <img
+              className="rounded-md object-cover h-full w-full"
+              src={existingImageUrl}
```

Comment:
<div>

<strong>Bito Code Review Agent Run</strong> <a href=https://github.com/omkardongre/Food-Ordering-App/pull/33#issuecomment-2394670085>#d60155</a><span> - 10/04/2024, 09:21 pm</span>
<br>

<div><small>🔴</small> High importance</div>

<div id="suggestion">
<div id="issue">
<b>Issue:</b> The img element lacks an alt attribute, which is crucial for accessibility. Screen readers and other assistive technologies rely on alt text to provide context for images.
</div>
<div id="fix">
<b>Fix:</b> Add an alt attribute to the img element that provides a concise description of the image content.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -28,6 +28,7 @@
              <img
                className="rounded-md object-cover h-full w-full"
                src={existingImageUrl}
 +              alt="Restaurant image"
              />
            </AspectRatio>
          )}
```
</div>
</details>
</div>
<br>


</div>

Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged